docs/content/en/docs/reference/clusterspec/irsa.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "IAM for Pods configuration"
+linkTitle: "IAM for pods"
+weight: 20
+description: >
+  EKS Anywhere cluster spec for Pod IAM (IRSA)
+---
+
+### IAM Role for Service Account on EKS Anywhere clusters with self-hosted signing keys
+
+IAM Role for Service Account (IRSA) enables applications running in clusters to authenticate with AWS services using IAM roles. The current solution for leveraging this in EKS Anywhere involves creating your own OIDC provider for the cluster, and hosting your cluster's public service account signing key. The public keys along with the OIDC discovery document should be hosted somewhere that AWS STS can discover it. The steps below assume the keys will be hosted on a publicly accessible S3 bucket. Refer [this](https://docs.aws.amazon.com/AmazonS3/latest/userguide/configuring-block-public-access-bucket.html) doc to ensure that the s3 bucket is publicly accessible.
```

Comment:
```suggestion
IAM Role for Service Account (IRSA) enables applications running in clusters to authenticate with AWS services using IAM roles. The current solution for leveraging this in EKS Anywhere involves creating your own OIDC provider for the cluster, and hosting your cluster's public service account signing key. The public keys along with the OIDC discovery document should be hosted somewhere that AWS STS can discover it. The steps below assume the keys will be hosted on a publicly accessible S3 bucket. Refer [this](https://docs.aws.amazon.com/AmazonS3/latest/userguide/configuring-block-public-access-bucket.html) doc to ensure that the S3 bucket is publicly accessible.
```
I think "S3" should be capitalized, unless you are specifically referring to the s3 command-line option.